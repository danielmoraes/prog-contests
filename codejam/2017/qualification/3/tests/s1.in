100
4 2
5 2
6 2
1000 1000
1000 1
1000 394
652 650
500 1
533 496
309 286
999 999
922 916
999 255
500 249
415 325
572 455
500 500
500 245
999 128
1000 127
821 804
5 1
653 531
999 511
1000 999
999 256
999 487
999 427
937 891
1000 255
500 254
865 664
500 2
1000 488
357 337
279 220
999 488
1000 498
500 128
4 1
1000 511
485 383
575 451
330 272
999 998
1000 512
570 466
389 386
580 574
1000 499
662 504
500 65
500 256
804 624
3 1
1000 256
573 472
1000 2
931 908
1000 500
1 1
500 116
337 300
863 668
999 499
426 342
701 625
999 2
500 244
999 498
500 127
887 801
2 1
500 499
672 525
1000 489
506 380
863 850
770 716
999 127
2 2
500 243
594 562
685 577
620 558
487 455
500 248
3 2
999 497
500 250
500 255
872 736
727 671
999 512
500 117
999 508
567 545
902 716
1000 128
999 1
