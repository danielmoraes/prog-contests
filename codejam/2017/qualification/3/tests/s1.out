Case #1: 1 0
Case #2: 1 0
Case #3: 1 1
Case #4: 0 0
Case #5: 500 499
Case #6: 1 1
Case #7: 0 0
Case #8: 250 249
Case #9: 0 0
Case #10: 0 0
Case #11: 0 0
Case #12: 0 0
Case #13: 3 2
Case #14: 1 0
Case #15: 0 0
Case #16: 0 0
Case #17: 0 0
Case #18: 1 0
Case #19: 3 3
Case #20: 7 6
Case #21: 0 0
Case #22: 2 2
Case #23: 0 0
Case #24: 1 0
Case #25: 0 0
Case #26: 1 1
Case #27: 1 1
Case #28: 1 1
Case #29: 0 0
Case #30: 3 2
Case #31: 1 0
Case #32: 0 0
Case #33: 125 124
Case #34: 1 1
Case #35: 0 0
Case #36: 0 0
Case #37: 1 0
Case #38: 1 0
Case #39: 1 1
Case #40: 2 1
Case #41: 1 0
Case #42: 0 0
Case #43: 0 0
Case #44: 0 0
Case #45: 0 0
Case #46: 0 0
Case #47: 0 0
Case #48: 0 0
Case #49: 0 0
Case #50: 1 0
Case #51: 0 0
Case #52: 3 3
Case #53: 0 0
Case #54: 0 0
Case #55: 1 1
Case #56: 1 1
Case #57: 0 0
Case #58: 250 249
Case #59: 0 0
Case #60: 1 0
Case #61: 0 0
Case #62: 3 3
Case #63: 0 0
Case #64: 0 0
Case #65: 1 0
Case #66: 0 0
Case #67: 0 0
Case #68: 249 249
Case #69: 1 1
Case #70: 1 0
Case #71: 3 2
Case #72: 0 0
Case #73: 1 0
Case #74: 0 0
Case #75: 0 0
Case #76: 1 0
Case #77: 0 0
Case #78: 0 0
Case #79: 0 0
Case #80: 7 6
Case #81: 0 0
Case #82: 1 1
Case #83: 0 0
Case #84: 0 0
Case #85: 0 0
Case #86: 0 0
Case #87: 1 0
Case #88: 0 0
Case #89: 1 0
Case #90: 1 0
Case #91: 1 0
Case #92: 0 0
Case #93: 0 0
Case #94: 0 0
Case #95: 3 2
Case #96: 1 0
Case #97: 0 0
Case #98: 0 0
Case #99: 3 3
Case #100: 499 499
